### DISCUSSION TITLE
5월 11일 수요일 오후 7시 저녁 모임

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#215] [General]
어제 온라인 모임에서 모이신
저, 동훈님, 우현님, 태형님이 모두 
다음주 수요일 저녁에 모이기로 했습니다.

장소는 오근내2닭갈고요.
저희 회사 근처입니다 (야호!)

http://naver.me/5y4qkVpL

참석하고 싶으신 다른 분들도 오실 수 있으면 좋을 것 같아요.

**Comment by momoci99** [comment]
갑툭튀해서 죄송합니다만 저도 살짝 껴도 될까용.. ㅎㅎㅎㅎㅎㅎ

**Reply by jongfeel** [reply]
저는 좋습니다.
일단 5명 자리로 예약을 하는 걸로 하겠습니다.

**Comment by jongfeel** [comment]
모임 사진 여기도 올려봅니다.

![image](https://user-images.githubusercontent.com/17442457/168187880-c9e457e7-d6ef-476c-933d-aa0e36b38867.png)

![IMG_0994](https://user-images.githubusercontent.com/17442457/168188013-112b2977-ad00-452e-a8f8-8fc35235d2f4.png)

![IMG_0995](https://user-images.githubusercontent.com/17442457/168188024-cc121d26-225c-4496-95fe-e8908a1e5967.png)
